index.html

```diff
@@ -205,46 +205,64 @@ <h1>About Myself</h1>
                 <button class="resume" type="button">Get my resume</button>
             </div>
 
-            <ul>
-                <li>                    
-                    <div class="skills">
-                    <div class="arrow">
-                        <p class="skill"> Languages</p>
-                         <p> <i class="fa fa-chevron-down" aria-hidden="true"></i> </p>
-                     </div> 
-
-                     <ul class ="languages">
-                        <li><img src="images/JavaScript.jpg" alt=""><a href="#" class = "tech">JavaScript </a></a></li>
-                         <li> <img src="images/html.jpg" alt=""><a href="#" class = "tech" >HTML5</a></li>
-                         <li><img src="images/css.jpg" alt=""><a href="#" class = "tech">CSS</a></li>
-                     </ul>
+            <div>
+                <ul>
+                    <li>                    
+                        <div class="skills">
+                        <div class="arrow">
+                            <p class="skill"> Languages</p>
+                            <p> <i class="fa fa-chevron-down" aria-hidden="true"></i> </p>
+                        </div> 
+
+                        <ul class ="languages">
+                            <li><img src="images/JavaScript.jpg" alt=""><a href="#" class = "tech">JavaScript </a></a></li>
+                            <li> <img src="images/html.jpg" alt=""><a href="#" class = "tech" >HTML5</a></li>
+                            <li><img src="images/css.jpg" alt=""><a href="#" class = "tech">CSS</a></li>
+                        </ul>
 
-                    </div>
-                </li>
+                        </div>
+                    </li>
+
+                    <li>
+                        <div class="arrow">
+                            <p class="skill">Frameworks</p>
+                            <p> <i class="fa fa-chevron-right" aria-hidden="true"></i></p>                            
+                        </div>
+                        <hr id="line-1">
+
+                    </li>
+
+                    <li>
+                        <div class="arrow"> 
+                            <p class="skill">Skills</p> <p> <i class="fa fa-chevron-right" aria-hidden="true"></i> </p>
+                        </div>
+                        <hr id="line-1">
+                    </li>
+                </ul>        
+            </div>
 
-                <li>
-                    <div class="arrow">
-                        <p class="skill">Frameworks</p>
-                        <p> <i class="fa fa-chevron-right" aria-hidden="true"></i></p>                            
-                    </div>
-                    <hr id="line-1">
+            <div class="form-container">
+                <img class="back-spot" src="./images/back-print.png" alt="" />
+                
+                <div class="form-bottom">
+                    <h1>Contact me</h1>
```

Comment:
- [ ] I would kindly suggest you make your `contact me` header another `hx` tag instead of `h1`. this is simply for accessibility and readability as it advisable and of a good practice to have only one `h1` header tag in a website.
You can make use of `h2` or `h3` 👍 👍

Reply:
Done